docs/05-rfcs/stories/story-11.md

```diff
@@ -82,11 +82,11 @@ resource TaskList {
     * @param term - the term to search
     * @returns set of task id that matches the term
     */
-  inflight find_tasks_with(term: str): Set<str> {
+  inflight find_tasks_with(term: str): Array<str> {
     print("find_tasks_with: ${term}");
     let task_ids = this.list_task_ids();
     print("found ${task_ids.size} tasks");
-    let output = MutSet<str>{};// #1172
+    let output = MutArray<str>[];// #1172
     for id in task_ids {
       let title = this.get_task(id); // https://winglang.slack.com/archives/C047QFSUL5R/p1674549602212669
       if title.contains(term) { 
```

Comment:
While you're at it - this line:
```
print("found ${output.size} tasks which match term '${term}'");
```
Please change to (`.len` instead of `.size`):
```
print("found ${output.len} tasks which match term '${term}'");
```